01-introduction.Rmd

```diff
@@ -1,6 +1,6 @@
 # Introduction {#intro}
 
-This book is about harnessing the power of modern computers to make the most of geographic data.
+This book is about harnessing the power of modern computers to 'do' exciting things with geographic data.
```

Comment:
Not sure the adverb 'exciting' helps here. Open to other ideas but to me keeping it simple and relatively objective in the opening sentences makes sense. I might find spatial aggregation exciting but someone else may find it boring. We should avoid making assumptions about the reader. They may be forced to read this for work.